converters/convertVol.cpp

```diff
@@ -57,11 +57,16 @@ using namespace DGtal;
   -o [ --output ] arg   volumetric file (.pgm3d, .vol, .longvol) 
 @endcode
 
-@b Example:
+@b Examples:
 @code 
 $ convertVol -i ${DGtal}/examples/samples/lobster.vol -o convertedVol.p3d 
 @endcode
 
+ To upgrade a "Version-2" vol file to a "Version-3" (default Vol writer):
+ @code
+ $ convertVol -i lobster.vol -o lobster.vol
```

Comment:
well... it was just an example.. not something the user could/should actually run...